lib/ssh/commands.go

```diff
@@ -20,14 +16,14 @@ type Cmd struct {
 }
 
 // RunCommands executes commands sequentially
-func RunCommands(ctx context.Context, logFn utils.LogFnType, client *ssh.Client, commands []Cmd) error {
+func RunCommands(ctx context.Context, node SshNode, commands []Cmd) error {
```

Comment:
minor: the acronyms are usually not camel-cased (Ssh vs SSH): https://github.com/golang/go/wiki/CodeReviewComments#initialisms